blacklisted_cidrs.yaml

```diff
@@ -0,0 +1,24 @@
+Schema: yaml_cidr
```

Comment:
Unsure if this matters much, but I've rarely seen uppercase in YAML keys (unless they're treated as strings instead of identifiers). Also, I'd suggest a file extension of `yml` rather than `yaml`, though this is purely preferential. Think about it like `jpg` vs `jpeg`.

Reply:
Yeah, I prefer `jpeg` (-:

Reply:
But we already have `rools.yml` and `users.yml`, so consistency becomes another point for me :-)

Reply:
Like I allude to in the updated PR description, I canged the Schema keys to capitalized to keep them at the top of the file. It seems that PyYAML forces alphabetical order and this is the simplest workaround, though I begrudge it.

Reply:
The consistency argument convinced me, though I note that this flies in the face of some official YAML recommendation.